experiments/slurm_base_char_sc.sh

```
@@ -0,0 +1,44 @@
+EXPERIMENT_PATH=../yaib_logs/${TASK_NAME}_experiment
+DATASET_ROOT_PATH= /sc-projects/sc-proj-cc08-cassandra/Prospective_Preprocessed/yaib_format #data/YAIB_Datasets/data
+
+echo "This is a SLURM job named" $SLURM_JOB_NAME "with array id" $SLURM_ARRAY_TASK_ID "and job id" $SLURM_JOB_ID
+echo "Resources allocated: " $SLURM_CPUS_PER_TASK "CPUs, " $SLURM_MEM_PER_NODE "GB RAM, " $SLURM_GPUS_PER_NODE "GPUs"
+echi "Task type:" ${TASK}
+echo "Task: " ${TASK_NAME}
+echo "Model: "${MODEL_NAME}
+echo "Dataset: "${DATASETS[$SLURM_ARRAY_TASK_ID]}
+echo "Experiment path: "${EXPERIMENT_PATH}
+
+cd ${YAIB_PATH}
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)